src/chrome/chromeDebugAdapter.ts

```diff
@@ -523,6 +557,13 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         if (this._initialSourceMapsP) {
             this._initialSourceMapsP = <Promise<any>>Promise.all([this._initialSourceMapsP, sourceMapsP]);
         }
+
+        const scriptEvent: ScriptEvent = this.scriptToScriptEvent(script);
+        if (this._scriptEventsBeforeInitializedEventFired !== null) {
```

Comment:
Is there a more descriptive way to test whether the initialized event has been fired yet?